videos/DdYC36N9z0E

```diff
@@ -0,0 +1,11 @@
+tags:
```

Comment:
Shouldn't video files have `.yml` extension?

Reply:
Yes, I should